datastore.go

```diff
@@ -75,7 +75,7 @@ func (d *Datastore) Query(q query.Query) (query.Results, error) {
 		q.Prefix = fmt.Sprintf("/%s", q.Prefix)
 	}
 	// thanks to petermattis for suggestion
-	// see https://github.com/petermattis/pebble/issues/168#issuecomment-507042838
+	// see https://github.com/cockroachdb/pebble/issues/168#issuecomment-507042838
```

Comment:
this should be changed back. The issue was opened against petermattis' repo, not cockroachdb repo